apps/src/codebridge/FileBrowser/hooks/useHandleFileUpload.ts

```diff
@@ -31,12 +22,8 @@ export const useHandleFileUpload = (
   const {newFile} = useCodebridgeContext();
   const dialogControl = useDialogControl();
   return useCallback(
-    ({folderId, fileName, contents}: handleFileUploadArgs) => {
-      // first of all, we just click on the document body to close our pop up
-      // this is because we canceled the original click event inside of the FileUploader
-      // of note - if additional clickhandlers were attached here, they won't be called.
-      // So don't attach other handlers to this button.
-      document.body.click();
+    (fileName: string, contents: string, folderIdArg: unknown) => {
+      const folderId = folderIdArg as FolderId;
```

Comment:
This function just needed its args re-ordered to play nice with the new `useFileUploader` hook.

However, the re-cast is silly. The hook expects a callback with the last arg typed as `unknown`, but it won't let me hand in the stricter function typed `FolderId`, so we re-cast it inside here.